configuration/init/systemd/tedge-mapper-collectd.service

```diff
@@ -4,9 +4,11 @@ After=syslog.target network.target mosquitto.service
 
 [Service]
 User=tedge-mapper
+Type=simple
```

Comment:
Is there any reason to change it to simple? I mean that `simple` is implied by using `ExecStart` anyway...